packages/editor/src/components/post-featured-image/index.js

```diff
@@ -43,7 +43,7 @@ const instructions = (
 	</p>
 );
 
-function getMediaDetails( media, postId ) {
+export function getMediaDetails( media, postId ) {
```

Comment:
This is specific function for featured image and the name is generic, because it was used internally by that component. I think it would be better to `lock` this an make it private instead of introducing a new API.

Reply:
Good call! Thanks for the nudge.